src/api/SearchApi.js

```diff
@@ -65,6 +66,48 @@ const SEARCH_TERM :string = 'searchTerm';
 const START :string = 'start';
 
 /**
+ * `GET /search/home/{start}/{maxHits}`
+ *
+ * Executes a search over all existing entity sets to populate the home page
+ *
+ * @static
+ * @memberof loom-data.SearchApi
+ * @param {number} start
+ * @param {number} maxHits
+ * @returns {Promise}
+ *
+ * @example
+ * SearchApi.loadHomePageEntitySets(0, 10);
+ */
+
+export function loadHomePageEntitySets(start :number, maxHits :number) :Promise<> {
```

Comment:
I'm not sure what to rename it -- the point of the new endpoint in loom-api was to load entity sets that should be shown on the home page. right now it's a generic search for all relevant (i.e. non association type) entity sets that you have permissions to, but eventually we might want to do the thing we had talked about where we have some sort of ranking for relevance/popularity. Do you have any other name ideas? It could be like "LoadAllEntitySetsPaginated" or something like that maybe?